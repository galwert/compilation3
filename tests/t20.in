int Compi(int hw3)
{
    return 100; // >=D
}

void main()
{
    int x = 1;
    
    if (true)
    {
        int t = 1;
    }
    else
    {
        int t = 1;
    }
    int t = 1;
    
    if (x) // Should fail here - int is not bool >=(
    {      // Please make sure you fail in line 20
        return;
    }
}

int Compu(int hw4)
{
    return 101; // >=D
}

bool Infi   (int hw13)
{
    return true;
}