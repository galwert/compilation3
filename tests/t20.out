---end scope---
hw3 INT -1
---end scope---
t INT 1
---end scope---
---end scope---
t INT 1
---end scope---
line 20: type mismatch
